@types/get-random-values/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'get-random-values' {
```

Comment:
This makes for easier testing in node, rather than using window.crypto,getRandomValue

Reply:
Historically, I believe we’ve avoided introducing external dependencies (especially ones without types) merely to improve the Primer developer experience (i.e. dependencies that do not _also_ improve the experience of Primer consumers and the experience of their end-users).

Follow-up questions:
1. Do we have established guidance about when to add third-party dependencies @colebemis @T-Hugs? (e.g. specific criteria dependencies must meet, specific situations when we’d avoid third-party solutions, guidelines for choosing between alternatives)
2. If `window.crypto.getRandomValue` is onerous, how many lines of code would it take to produce a utility function which provides a nicer API but uses `window.crypto.getRandomValue` internally?

Reply:
It's such a simple method, I went ahead and just replaced it with a util that will generate a random string 👍